core/domain/suggestion_services.py

```diff
@@ -165,6 +166,20 @@ def _update_suggestion(suggestion):
     suggestion_model.put()
 
 
+def edit_suggestion(suggestion_id, new_change):
+    """Edits the submitted suggestion.
+
+    Args:
+        suggestion_id: obj. Id of suggestion to be changed.
+        new_change: dict(str, str). Dictionary contains the new suggestion.
+    """
+    suggestion = get_suggestion_by_id(suggestion_id)
+    old_change = suggestion.change.to_dict()
+    old_change = new_change
```

Comment:
I didn't understand the use case from line 176 to 178?

Reply:
here 
```get_suggestion_by_id(suggestion_id)``` return suggestion object from model.
``` suggestion.change.to_dict() ``` returns the change of suggestion  (change means the suggestion or say what we wan't to change in exploration) in dictionary format
``` old_change = new_change ``` represent that new suggestion (or new change in exploration) is assigned to old suggestion (or old change in exploration).
thanks